packages/cli/src/ceramic-logger-plugins.ts

```diff
@@ -117,7 +241,7 @@ export class LogToFiles {
      */
     private static async _rotate (filePath: string): Promise<void> {
         try {
-          await fsPromises.rename(filePath, `${filePath}.old`)
+            await fsPromises.rename(filePath, `${filePath}.old`)
```

Comment:
what happens after the second rotation?  Don't you wind up overriding your old `<filename>.old` log file?  I think we'd be better off encoding a timestamp into the file name for historical log files, so we can keep around all historical logs until they are manually archived.

Reply:
Yes to keep this simple we are overwriting. These logs do get archived by Loki after we ship them so we don't need to add that complexity here.

Reply:
How quickly/frequently do they get shipped/backed up?  Is there a risk if we fill a file too quickly that we might rotate over it before it's backed up?

Reply:
they are getting shipped immediately (on every write). so i don't think there should be an issue of a file getting filled too quickly.

Reply:
Thinking about this more, that solution only works for our infra, not for others running their own js-ceramic nodes.  Does it add a lot of complexity to encode a timestamp into the file name?  That's how I've usually seem log rotation handled so that we never override an existing log.  That leaves more flexibility for different deployments to use different archiving strategies.

Reply:
Hmm... i mean we can add a time stamp but that will won't be proper rotation. Logs will just grow infinitely. We would need compression at least too and some removal strategy. It's a bit of a can of worms. Using the [package I mentioned](https://www.npmjs.com/package/rotating-file-stream) will help to take care of this so I'll add it as a story for this sprint. For this pr though I'd rather not because it is currently broken :sweat_smile:

Reply:
@stbrody seem ok to you?

Reply:
Yeah, the point is separation of concerns.  js-ceramic shouldn't mandate any particular archival strategy, and we shouldn't be performing log cleanup internally as we could wind up deleting logs that are still needed.  From the js-ceramic perspective, yes, that would mean that the size of the logs grow unbounded, but there could be external processes to archive and remove old logs so we don't fill up the storage volume.  I do feel kind of strongly that js-ceramic shouldn't be deleting logs out from under our users, but I don't think it needs to be a release blocker.  If you want to merge this and move on building our infrastructure with this log rotation strategy for now, that's fine, but I do think we'll want to revisit this not too long after launch so that users who are running their own nodes have flexibility in how they handle log archival.

Reply:
Agree!